.github/workflows/build.yaml

```diff
@@ -0,0 +1,51 @@
+name: Build
+
+on:
+- pull_request
+- push
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install build dependencies
+        run: |-
+          sudo apt-get update
+          sudo apt-get install --yes --no-install-recommends \
+            bison \
+            doxygen \
+            flex \
+            gettext \
+            graphviz \
+            liborc-0.4-dev \
+            libpng-dev \
+            libasound2-dev \
+            libsdl1.2-dev \
+            libgl1-mesa-dev \
+            pkg-config
+
+      - name: Checkout Git branch
+        uses: actions/checkout@v2.0.0
+
+      - name: Run CMake
+        run: |-
+          mkdir build
+          cd build
+          cmake ../libvisual/ \
+            -DENABLE_ALSA=yes \
+            -DENABLE_DANCINGPARTICLES=yes \
+            -DENABLE_DOCS=yes \
+            -DENABLE_GOOM2K4=yes \
+            -DENABLE_NLS=yes
```

Comment:
Are the tests working? It might be a good idea to add compiling the tests and run them if possible

Reply:
Good point!

I think I'll merge it without tests first, because in the repo GitHub Actions is not picking it up for some reason. I hope the merge will fix that. it worked well over here:
https://github.com/hartwork/libvisual/commit/40ef05e89442ab834db26a6b1fa18183b7d9efe2/checks?check_suite_id=370815364

Reply:
I'm running the test suite now but it's failing. I'll try to have it run but ignore the exit code for now so we can actually merge it.

Reply:
Sounds good